dev.robiot.xclicker.yml

```diff
@@ -0,0 +1,33 @@
+app-id: dev.robiot.xclicker
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: xclicker
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  # - --device=all
+  - --filesystem=~/.config
```

Comment:
Can you explain what is it needed for?

Reply:
You marked my questions as resolved but I don't think I've seen a reply to these.

Reply:
ping @robiot

Reply:
The flatpak only needs to access `./xclicker.conf`, so I think this can be updated to ` - --filesystem=~/.config/xclicker.conf `

Reply:
Yeah

Reply:
this doesn't work for files sadly. Ideally it should conform to the xdg specification by querying your toolkit for the config directory.

Reply:
Yeah; you get write access to $XDG_CONFIG_HOME by default; if it's not defined, you want to fall back to $HOME/.config.

Reply:
I suppose there isn't anything that needs a change here so I am going to resolve this. Correct me if i'm wrong.